mon-pix/tests/acceptance/authentication/login_test.js

```diff
@@ -0,0 +1,72 @@
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { click, currentURL } from '@ember/test-helpers';
+import { visit } from '@1024pix/ember-testing-library';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import setupIntl from '../../helpers/setup-intl';
+
+module('Acceptance | Login', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+  setupIntl(hooks, ['fr', 'en']);
+
+  module('International domain (.org)', function () {
```

Comment:
**thought**
Il n'y a rien dans ce test qui simule le fait qu'on soit sur un domain en .org. Je pense que ça passe parce que par défaut, `currentDomain.isFranceDomain` doit être `false` ou `undefined`. Du coup, je serai soit pour trouver un moyen de tester réellement les deux domaines, soit retirer ce `module` qui est trompeur.

Reply:
Bon, sans surprise, je me suis cassé les dents comme vous et je n'ai pas trouvé de bonne solution. 
Je n'aime pas trop l'idée de mettre la condition liée au domaine dans le composant, car ça implique d'injecter le service `currentDomain` et ça rend le composant moins "dumb". Ça ne relève pas de sa responsabilité, il me semble. 
Faute de mieux, je propose de mettre un titre plus explicite au module du style "when domain does not have .fr tld", ce qui correspond au cas réel (.org) comme au cas testé (localhost).

Reply:
+1 pour ne pas mettre la logique métier dans le composant